### PR TITLE
Use testcontainers as runner for elasticsearch #1014 

### PR DESCRIPTION
**Pull request by farodin91**
This commit contains:
* Update elasticsearch client lib to 6.6.0
* Removal of internal elasticsearchrunner
* Tinkerpop dependency management is moved into the root pom.xml

Part of #1014 

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by farodin91** [comment]
@chupman Do you like to review?

**Comment by chupman** [comment]
@farodin91 I'll try to take a look this week, but I'm not positive I'll find the cycles.

File: pom.xml [hunk 1]
```diff
@@ -481,6 +483,91 @@
...
+            <dependency>
+                <groupId>org.apache.tinkerpop</groupId>
+                <artifactId>tinkergraph-gremlin</artifactId>
+                <version>${tinkerpop.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.tinkerpop</groupId>
+                <artifactId>hadoop-gremlin</artifactId>
+                <version>${tinkerpop.version}</version>
+                <exclusions>
```
[the remark this answers is not in the document]
Comment (farodin91) [reply pom.xml]: I will check these exclusions.

**Comment by farodin91** [comment]
I will rebase on top of 0.3.
@chupman Do you want to review?

**Comment by chupman** [comment]
I don't have the cycles to do a review right now. Please don't wait on me.

**Pull request merged**